src/App/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.scss';
+import { Homepage } from '../Homepage/Homepage'
+
+function App() {
+  return (
```

Comment:
We're going to need to use `Router` here in the return, just for future reference.